gramlib/grammar.ml

```diff
@@ -1593,6 +1593,7 @@ module Parsable = struct
     | Stream.Error _ as exc ->
       let loc = get_loc () in
       Loc.raise ~loc exc
+    | Sys.Break -> raise Sys.Break (* No location on Sys.Break *)
```

Comment:
More generally, shouldn't we protect against all asynchronous exceptions?

Reply:
Do you mean `Out_of_memory` and `Stack_overflow`? Maybe it is ok to leave them with a location attached to them, so as to explain what token generated them?

Reply:
Also in this code we should use `iraise` I think, the original use of Loc.raise was due to camlp5 not having access to Coq machinery IIRC.

Reply:
You mean `Exninfo.iraise (Exninfo.capture Sys.Break)`??